app/controllers/jobseekers/job_applications/qualifications_controller.rb

```diff
@@ -101,7 +85,11 @@ def qualification
     @qualification ||= job_application.qualifications.find(params[:id])
   end
 
+  def qualifications
+    @qualifications ||= [job_application.qualifications.find(params[:ids])].flatten
```

Comment:
what is the difference between:
`[job_application.qualifications.find(params[:ids])].flatten`
and just
`job_application.qualifications.find(params[:ids])`

Reply:
This happened because, when I passed in a single id, `find` returned a qualification, not an array, which broke things like `.count`.

I will fix this by passing the single id in an array, since `.find([id])` returns an array `[qualification]`.

Reply:
Changes pushed

Reply:
ah OK, now I get it. if you want to always be a collection, you can do:
`job_application.qualifications.where(id: params[:ids])`
that will always return a collection if you pass a string or an arrays, and later you can do:
`qualifications.destroy_all` 
instead of 
`qualifications.each(&:destroy)`

Reply:
Will fix in next PR.